edge/pkg/common/message/message_test.go

```diff
@@ -0,0 +1,100 @@
+package message_test
+
+import (
+	"testing"
+
+	"github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/message"
+	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion

	"github.com/stretchr/testify/assert"
	
	"github.com/kubeedge/beehive/pkg/core/model"
	"github.com/kubeedge/kubeedge/edge/pkg/common/message"
	
```
The package start with `github.com/kubeedge` should be placed in the third part.